src/Worker.Extensions.DurableTask/FunctionsDurableClientProvider.net6.cs

```diff
@@ -29,6 +29,7 @@ private static GrpcChannel CreateChannel(ClientKey key)
         {
             HttpClient = httpClient,
             DisposeHttpClient = true,
+            MaxReceiveMessageSize = maxGrpcMessageSize
```

Comment:
Does this need to be a header sent back to the host?

Reply:
Sorry I don't quite follow this comment too. I think this file, FunctionsDurableClientProvider won't be sent back to the host? It's just stay in worker.extension.durabletaks.

Reply:
oh nvm, thought this was the gRPC header block being added.